src/interpreter_funcs/voxelize.rs

```diff
@@ -12,12 +12,13 @@ use crate::interpreter::{
 };
 use crate::mesh::voxel_cloud::VoxelCloud;
 
-const VOXEL_COUNT_THRESHOLD: u32 = 50000;
+const VOXEL_COUNT_THRESHOLD: u32 = 100_000;
```

Comment:
We can afford more on the NUC, which is our reference hardware. Actually, we can afford even more than this (10M voxel operations take about 15s).